examples/cellfinder_cell_density.py

```diff
@@ -13,38 +13,55 @@
 import brainglobe_heatmap as bgh
 
 # get the number of cells for each region
-cells_path = Path(__file__).parent / "cell_counts_example.h5"
-
-data = pd.read_hdf(cells_path)
-cell_counts = data.groupby("region").count()
+cells_summary = pd.read_csv(Path(__file__).parent / "summary.csv")
 
 # get regions two levels up the hierarchy
 atlas = BrainGlobeAtlas("allen_mouse_25um")
-parent_regions = [
-    atlas.get_structure_ancestors(r)[-2] for r in cell_counts.index
-]
+structures_df = atlas.lookup_df
 
-cell_counts["parent_region"] = parent_regions
-cell_counts = cell_counts.groupby("parent_region").sum()
+# Merge the cells summary with the structures CSV to get structure
+# acronyms and IDs
+cells_summary = pd.merge(
+    cells_summary, structures_df, left_on="structure_name", right_on="name"
+)
 
+# Fetch the 2nd order parent region for each region to accumulate
+# cell counts at that level
+cells_summary["parent_region"] = [
+    (
+        atlas.get_structure_ancestors(r)[-2]
+        if (len(atlas.get_structure_ancestors(r)) > 1)
+        else None
+    )
+    for r in cells_summary["id"]
+]
+# Filter out rows where parent_region is NaN
+cells_summary = cells_summary[cells_summary["parent_region"].notna()]
 
-# get regions' volume
+# Accumulate cell counts for each parent region
+cells_summary = cells_summary.groupby("parent_region", as_index=False).sum(
+    numeric_only=True
+)
+cells_summary.set_index("parent_region", inplace=True)
+
+# Get regions' volume
 volumes = []
-for region in cell_counts.index:
+for region in cells_summary.index:
     obj_file = str(atlas.meshfile_from_structure(region))
     mesh = load_mesh_from_file(obj_file)
     volumes.append(mesh.volume())
 
-# get the density (num cells / volume)
-cell_counts["volume"] = volumes
-cell_counts["density"] = cell_counts["x"] / cell_counts["volume"]
-cell_counts = cell_counts.loc[cell_counts.density > 5 * 1e-9]
-
-print(cell_counts)
+# Calculate the density using atlas volume for each region
+cells_summary["volume_mm3"] = volumes
+cells_summary["cells_per_mm3"] = (
+    cells_summary["total_cells"] / cells_summary["volume_mm3"]
+)
 
+# Filter out regions with very low cell density
+cells_summary = cells_summary.loc[cells_summary["cells_per_mm3"] > 5 * 1e-9]
```

Comment:
```suggestion
cells_summary = cells_summary.loc[cells_summary["cells_per_mm3"] > 5 * 1e-9]

print(cells_summary[["volume_mm3","cells_per_mm3"]])
```
Unless getting rid of the print statement was intentional?